config.go

```diff
@@ -65,11 +66,12 @@ func CreateConfig(path string) error {
 			config.LoginHost = "my.rightscale.com"
 		}
 		hostDef = fmt.Sprintf(" (%v)", config.LoginHost)
+		refreshTokenDef = fmt.Sprintf(" (%v)", config.RefreshToken)
```

Comment:
Also encrypt the refresh token instead of writing as plain text similar to the password?